app/src/main/java/module2/globant/calculator/Constants/Constants.java

```diff
@@ -0,0 +1,10 @@
+package module2.globant.calculator.Constants;
+
+public class Constants {
+
+    public static final int SUM = 1;
```

Comment:
you could create an enumerated int definition 
https://developer.android.com/reference/android/support/annotation/IntDef